server/handlers/requestHandlers.go

```diff
@@ -6,8 +6,18 @@ import (
 	"net/http"
 	"server/middleware"
 	"server/utils"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
 )
 
+var jwtKey = []byte("secret_key")
+
+type Claims struct {
+	Username string `json:"username"`
```

Comment:
Can you place this struct in models.go in middleware? Also name the field "email" instead of "Username", use the json tag, "email", and add the bson tag, "email".